examples/chat_app/requirements.txt

```diff
@@ -0,0 +1,11 @@
+gradio==3.42.0 # Gradio 3.47 breaks the UI and versions between 3.42 and 3.47 haven't been tested
+huggingface-hub
+markdown
+mdtex2html
+neural-compressor
```

Comment:
do you need neural-compressor

Reply:
Deleted both neural-compressor and optimum